src/lib/chat/matrix/chat-message.ts

```diff
@@ -80,3 +80,15 @@ export function mapEventToAdminMessage(matrixMessage) {
     admin: adminData,
   };
 }
+
+export function mapEventToBrowserNotificationMessage(matrixMessage) {
```

Comment:
`mapToLiveRoomEvent` or something? The event we fire is `receiveLiveRoomEvent` and this level doesn't know if this is going to be used for browser notifications or anything else.